server/storage/sql_models.go

```diff
@@ -17,6 +21,20 @@ func (g TUFFile) TableName() string {
 	return "tuf_files"
 }
 
+// Change defines the the fields required for an object in the changefeed
+type Change struct {
+	ID        uint `gorm:"primary_key" sql:"not null"`
+	CreatedAt time.Time
+	GUN       string `gorm:"column:gun" sql:"type:varchar(255);not null"`
+	Version   int    `sql:"not null"`
+	Sha256    string `sql:"type:varchar(64);"`
```

Comment:
We might want to consistent whether to use SHAXXX or ShaXXX, we use both at present. But it can be totally leave to another separated PR.